src/server/routes/deploy/published.ts

```diff
@@ -88,7 +87,6 @@ export async function deployPublished(fastify: FastifyInstance) {
       });
 
       reply.status(StatusCodes.OK).send({
-        deployedAddress,
```

Comment:
imo leave this behavior

Reply:
reverted changed and added `message` to send in response if the `deployedAddress` cannot be resolved